### DISCUSSION TITLE
PC requirements for DIA-NN

### DISCUSSION THREAD

**Discussion by drames29** [vdemichev/DiaNN#1292] [General]
My lab is looking to purchase a proteomics PC specifically for DIA-NN analysis. Previously we had been using a PC with just 8 logical processors, which lead to massively long processing times.

As someone who doesn't have a great knowledge of computing, what sort of PC would be best for running DIA-NN, and what specs, other than number of logical processors, are most important.

Any help would be appreciated, thanks.

**Comment by vdemichev** [comment]
Hi,

AMD 7950X if you can get it with 96Gb or (better) 128Gb RAM is quite good in terms of speed/cost. 
If budget allows - Threadripper.

Best,
Vadim
